src/plugins/cache/README.md

```diff
@@ -5,7 +5,7 @@
 - infos/provides =
 - infos/recommends =
 - infos/placements = pregetcache postgetcache
-- infos/status = maintained unittest shelltest specific global
+- infos/status = maintained unittest shelltest specific nodep global
```

Comment:
@markus2330 do we really want `./scripts/dev/update-infos-status --auto`?

Seems like it updates quite a few things. Here for instance it added nodep because no `find_package` or `LINK_LIBRARIES` was found. Afaik nodep means only a C99 compiler suffices, which is not the case. Cache needs a posix compliant system.

At other places it removed memleak status etc., it seems quite suspicious to me.

Reply:
I think in the long term we want `--auto` but I see it as task in #3590 to fix the wrong heuristics. For the current release I think we should not use it. Adding "nodep" for "cache" is the only change that sounds okay to me without further investigation. But I also agree with not having "nodep", as it needs POSIX, so it is borderline.